### ISSUE TITLE
Node points of chart which are accessible with mouse are not keyboard accessible.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#20808]
#### Expected behaviour
Node points of chart which are accessible with mouse should be keyboard accessible.

#### Actual behaviour
 Node points of chart which are accessible with mouse are not keyboard accessible.
  **Observation:** By invoking the node point with keyboard, unexpected error is occurred under console however by invoking the node point with mouse, we are getting the result for the node point.

#### Live demo with steps to reproduce

1.Open URL: https://jsfiddle.net/n8ohxq7m/2/show. 
2.Press TAB key and navigate to the first node point. Now press arrow keys to navigate the node points.
3.Verify whether the same value is fetched under the console by invoking the node point with keyboard and mouse or not.

[MAS2.1.1_Node points of chart which are accessible with mouse are not keyboard accessible. (1).webm](https://github.com/highcharts/highcharts/assets/93735775/d5e815df-8f41-49f9-a7c9-02c33ff57fb3)

#### Product version

Highcharts

#### Affected browser(s)
OS: Windows 11 23H2 (OS Build:22631.3007)

**Comment by oysteinmoseng** [comment]
Thank you for reporting. It seems we are not passing in `event.point` for the click events performed with keyboard.

**Comment by oysteinmoseng** [comment]
Closing as duplicate of https://github.com/highcharts/highcharts/issues/20729
